packages/client/src/createClient.ts

```diff
@@ -71,10 +70,11 @@ export function createClientInternal(
   }
 
   const client: Client = Object.defineProperties<Client>(
-    clientFn as UnstableClient,
+    clientFn as Client,
     {
       ctx: {
         value: clientCtx,
+        writable: false,
```

Comment:
Why did we need to add this?

Reply:
I was trying to protect caches a bit better but I think this is superseeded by other changes I made for the omniapi anyway.